resources/UserActions.ts

```diff
@@ -0,0 +1,42 @@
+import { UserData } from './UserData';
+const bcrypt = require('bcrypt');
+
+export class UserActions extends UserData {
+    private _PASSWORD_HASH_SALT = 10;
+    
+    //just in case we need the salt value somewhere else in the future
+    public get salt() {
```

Comment:
Is this method actually utilized? If not, consider removing it. We aim to avoid having unused code, relying on version control (like Git) to track our changes.

Reply:
Only once, inside hashPassword()